jaxrs-spec/src/main/asciidoc/chapters/providers/_lifecycle_and_environment.adoc

```diff
@@ -63,11 +62,10 @@ have a public constructor for which the JAX-RS runtime can provide all
 parameter values. Note that a zero argument constructor is permissible
 under this rule.
 
-A public constructor MAY include parameters annotated with
-`@Context`—Chapter <<context>> defines the parameter types permitted for
-this annotation. Since providers may be created outside the scope of a
-particular request, only deployment-specific properties may be available
-from injected interfaces at construction time; request-specific
+A public constructor MAY include injected parameters, provided the
+parameter type is a CDI bean. Since providers may be created outside the
+scope of a particular request, only deployment-specific properties may
+be available from injected interfaces at construction time; request-specific
```

Comment:
This seems fine, but it's all governed by CDI now, so perhaps we don't need to be so explicit anymore.

Reply:
I am pretty sure I used this comment about the unavailable injection of objects outside of the request scope when explaining this to a customer. But it would be likely mentioned in the CDI Spec as well.